assets/tests/factories/stop.ts

```diff
@@ -1,15 +1,19 @@
 import { Factory } from "fishery"
 import { LocationType } from "../../src/models/stopData"
 import { Stop } from "../../src/schedule"
+import { localGeoCoordinateFactory } from "./geoCoordinate"
 
-const stopFactory = Factory.define<Stop>(({ sequence }) => ({
-  id: `stop${sequence}`,
-  name: `Some Stop - ${sequence}`,
-  locationType: LocationType.Stop,
-  vehicleType: 3,
-  lat: 0,
-  lon: 0,
-  routes: undefined,
-}))
+const stopFactory = Factory.define<Stop>(({ sequence }) => {
+  const coord = localGeoCoordinateFactory.build()
+  return {
+    id: `stop${sequence}`,
+    name: `Some Stop - ${sequence}`,
+    locationType: LocationType.Stop,
+    vehicleType: 3,
+    lat: coord.latitude,
+    lon: coord.longitude,
```

Comment:
@firestack I really appreciate the `localGeoCoordinateFactory` you added! Using it here at the `stopFactory` seemed like the minimal change to get tests working now that we only render a single stop marker when stop markers are co-located. Interested if you have any concerns about this approach!

Reply:
I looked this over and it looks good to me!
I think we can extend this in the future if we need, so this looks like a great addition 👍🏻